indra/tests/test_cbio_client.py

```diff
@@ -51,3 +56,20 @@ def test_get_mutations_ccle():
     assert 'I208V' not in muts['A101D_SKIN']['BRAF']
     assert len(muts['LOXIMVI_SKIN']['AKT1']) == 0
     assert len(muts['A101D_SKIN']['AKT1']) == 0
+
+
+def test_get_profile_data():
+    profile_data = cbio_client.get_profile_data(cbio_client.ccle_study,
+                                                ['BRAF', 'PTEN'],
+                                                'COPY_NUMBER_ALTERATION',
+                                                'all')
+    assert profile_data['BT20_BREAST']['PTEN'] == -2
+    assert profile_data['BT20_BREAST']['BRAF'] == 1
+    assert profile_data['LOXIMVI_SKIN']['PTEN'] == 0
+    assert profile_data['LOXIMVI_SKIN']['BRAF'] == 0
+
+
+def test_get_ccle_cna():
+    profile_data = cbio_client.get_ccle_cna(['BRAF'])
+    assert profile_data['BT20_BREAST']['BRAF'] == 1
+    assert profile_data['LOXIMVI_SKIN']['BRAF'] == 0
```

Comment:
This test only works because of the hard-coded parameters in get_ccle_cna

Reply:
- Fixed hardcoded list in function with commit a610f4c
- Updated tests in commit 5860b83